developer-extension/src/panel/components/tabs/infosTab.tsx

```diff
@@ -169,7 +169,7 @@ function Entry({
   }
 
   const handleClearClick = () => {
-    onChange && onChange(null)
+    onChange!(null)
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏ `onChange` can be undefined
```suggestion
    onChange?.(null)
```